emr/application/views/main/sidebar.php

```diff
@@ -1,19 +1,17 @@
-  <body class="nav-md">
+<body class="nav-md">
     <div class="container body">
       <div class="main_container">
         <div class="col-md-3 left_col">
           <div class="left_col scroll-view">
             <div class="navbar nav_title" style="border: 0;">
-              <a href="main/index" class="site_title"><img src="../img/rsz_1certuslogowatermark.png"><span>Certus</span></a>            
+                <a href="main/index" class="site_title"><span><img src="../img/MGH.svg" style="padding: 0 0 10px 40px; "></span></a>
```

Comment:
I think it looks better with 50px instead of 40px

Reply:
Changed

Reply:
Looks good.